mlir/include/mlir/Dialect/MIOpen/sqlite_db.h

```diff
@@ -0,0 +1,150 @@
+//===------- sqlite_db.h - MLIR sqlite database client ----------===//
```

Comment:
should this interface be stored under `Tuning` directory?

Reply:
Will do